marko.json

```diff
@@ -385,7 +386,16 @@
     "@a11y-heading-text": "string",
```

Comment:
Seems like heading-text is a no-op in the case of the default section type (aka light guidance), because it relies on an icon. I think this will also break the aria-labelledby connection. In this case we should use aria-label instead of aria-labelledby. It's getting a bit messy, but will ultimately be cleaned up when I refactor the HTML side, and we can always use aria-labelledby.

Reply:
Done, I changed it so that if it's light guidance then it will set `aria-label` with the value of  `a11y-heading-text`. `aria-labelledby` will be null since status `widget.elId("status")` will be null/undefined